src/schemas/Record.js

```diff
@@ -34,6 +34,10 @@ const schema = new mongoose.Schema({
 		minlength: 2,
 		maxlength: 2
 	},
+	clientCountry: {
+		type: String,
+		required: false
+	},
```

Comment:
You don't have to put the required, it is by default false.
```suggestion
	clientCountry: {
		type: String
	},
```